src/binary/merkle_tree.rs

```diff
@@ -275,13 +385,15 @@ mod test {
 
     #[test]
     fn load_returns_a_valid_tree() {
-        const LEAVES_COUNT: u64 = 7;
+        const LEAVES_COUNT: u64 = (1 << 16) - 1;
```

Comment:
Would it be clearer to just use a number?

Reply:
I'm choosing to use a number that is a power of 2 minus 1 because under that scenario, we have the greatest number of MMR peaks possible. I can replace the left shift with a `.pow(16)` for increased readability. Ultimately, the leaves count is arbitrary and its concrete value doesn't actually matter for the purposes of this test.